lab-keiran/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "lab-keiran",
+  "version": "1.0.0",
+  "description": "",
+  "main": "cowsay.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
You forgot to include 'gulp' and 'gulp-eslint' as devDependencies, so gulp threw errors when I tried to run it.